go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/tidepool-org/tide-whisperer
 
 go 1.12
 
-replace github.com/tidepool-org/go-common => github.com/mdblp/go-common v0.3.0
+replace github.com/tidepool-org/go-common => github.com/mdblp/go-common v0.3.1-0.20200416150959-69aa19e1fce6
```

Comment:
Mystery of go.mod. The 69aa19e1fce6 references the good commit ID in go-common repository... But takes it as V0.3.1 (maybe because our tag dbl.0.4.0 doens't respect semVer).

But It also works with 
`replace github.com/tidepool-org/go-common => github.com/mdblp/go-common vdblp.0.4.0
`